source/plugins/logging.rst

```diff
@@ -2,14 +2,77 @@
 Logging and Debug Messages
 ==========================
 
-Java comes with a few logging frameworks. Logging is preferrable to printing to stdout (or stderr) with ``System.out.println()`` for a number of reasons:
+There are a few logging frameworks available for use in Java. Logging is preferable to printing to stdout or stderr with ``System.out.println()`` for a number of reasons:
 
-* Logged messages have a source name attached to them so it is possible to figure out where the logged messages are coming from.
-* Logged messages have a severity level which allows for simple filtering (i.e. disable all non-critical notices).
+* Logged messages are labeled with a source name, making it easier to figure out where the logged messages are coming from.
+* Logged messages have a severity level which allows for simple filtering (e.g. disable all non-critical notices).
 * The available logger frameworks allow you to enable or disable messages from certain sources.
 
+Sponge uses ``org.slf4j.Logger``, not ``java.util.logging.Logger``.
+
 Getting a Logger
 ================
 
+The Guice module used during the initialization of plugins has a plugin-scoped logger. This allows you to annotate a Logger field or object with ``@Inject`` to get the logger for your plugin, which is pre-configured with the correct plugin ID.
+
+.. note::
+    See :doc:`quick-start` for information on configuring your plugin ID.
+
+**Example - Field**
+
+.. code-block:: java
+
+    import com.google.inject.Inject;
+    import org.slf4j.Logger;
+
+    @Inject
+    private Logger logger;
+
```

Comment:
it is also possible to inject it in the constructor, that might be worth mentioning